src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -703,7 +703,7 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
     needRevert = false;
 
     // We might have a new operational identity, so we should start advertising it right away.
-    app::DnssdServer::Instance().AdvertiseOperational();
+    LogErrorOnFailure(app::DnssdServer::Instance().AdvertiseOperational());
```

Comment:
I noticed it started before your PR but we shouldn't really use logging that refers to `__FILE__` in the code for constrained devices. In general, I don't know why LogErrorOnFailure includes the error location while we already have `CHIP_CONFIG_ERROR_SOURCE` exactly for that. I think if we want to start using these macros more widely, LogErrorOnFailure should not directly refer to `__FILE__` and `__LINE__` macros (and rely on `CHIP_CONFIG_ERROR_SOURCE`).